seed/challenges/02-javascript-algorithms-and-data-structures/basic-javascript.json

```diff
@@ -1133,8 +1133,9 @@
       ],
       "tests": [
         "assert(!/\\\\/g.test(code) && myStr === '<a href=\"http://www.example.com\" target=\"_blank\">Link</a>', 'message: Remove all the <code>backslashes</code> (<code>\\</code>)');",
-        "assert(code.match(/\"/g).length === 4 && code.match(/'/g).length === 2, 'message: You should have two single quotes <code>&#39;</code> and four double quotes <code>&quot;</code>');"
+        "assert((function() { const strict = \"'use strict';\"; const tail = '(function() { return \"myStr = \" + myStr; })();'; const sliced = code.slice(code.indexOf(strict) + strict.length, code.indexOf(tail)); if (sliced.match(/\"/g).length === 4 && sliced.match(/'/g).length === 2) return true; return false; })() === true, 'message: You should have two single quotes <code>&#39;</code> and four double quotes <code>&quot;</code>');"
```

Comment:
We also have to consider `const` in the test `IIFE`. Can we safely assume that all environments that will run this challenge can handle ES6? 

Most semi-modern browsers can, but I'm not sure we want to break backwards compatibility with older browsers.

Even though it hurts (Gah, `var`!?), I suggest we replace `const` with `var`.